analyzers/src/SonarAnalyzer.Common/Rules/AspNet/BackslashShouldBeAvoidedInAspNetRoutesBase.cs

```diff
@@ -42,7 +44,8 @@ public abstract class BackslashShouldBeAvoidedInAspNetRoutesBase<TSyntaxKind> :
 
     protected void Check(SonarSyntaxNodeReportingContext c)
     {
-        if (Language.Syntax.NodeExpression(c.Node) is { } expression
+        if (!IsNamedAttributeArgument(c.Node)
+            && Language.Syntax.NodeExpression(c.Node) is { } expression
             && Language.FindConstantValue(c.SemanticModel, expression) is string constantRouteTemplate
```

Comment:
We aslodo have the `TryGetAttributeValue` extension method for such extractions.